synapse/config/redis.py

```diff
@@ -54,4 +60,16 @@ def generate_config_section(self, **kwargs: Any) -> str:
           # Optional password if configured on the Redis instance
           #
           #password: <secret_password>
+
+          # Optional one or more Redis hosts to use for long term sharedd caches.
```

Comment:
Just noticed a typo while reading through this out of interest. 😄 

```suggestion
          # Optional one or more Redis hosts to use for long term shared caches.
```